phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,68 @@
-<!-- INCLUDE ucp_header.html -->
-
-<!-- IF not PROMPT -->
-	<!-- INCLUDE ucp_pm_message_header.html -->
-<!-- ENDIF -->
-
-<!-- IF PROMPT -->
-	<h2>{L_EXPORT_AS_CSV}</h2>
-	<form id="viewfolder" method="post" action="{S_PM_ACTION}">
-	<div class="panel">
-		<div class="inner">
-		<h3>{L_OPTIONS}</h3>
-		<fieldset>
-			<dl>
-				<dt><label for="delimiter">{L_DELIMITER}{L_COLON}</label></dt>
-				<dd><input class="inputbox" type="text" id="delimiter" name="delimiter" value="," /></dd>
-			</dl>
-			<dl>
-				<dt><label for="enclosure">{L_ENCLOSURE}{L_COLON}</label></dt>
-				<dd><input class="inputbox" type="text" id="enclosure" name="enclosure" value="&#034;" /></dd>
-			</dl>
-		</fieldset>
-		</div>
-	</div>
-	<fieldset class="submit-buttons">
-		<input type="hidden" name="export_option" value="CSV" />
-		<input class="button1 button button-form" type="submit" name="submit_export" value="{L_EXPORT_FOLDER}" />&nbsp;
-		<input class="button1 button button-form-bold" type="reset" value="{L_RESET}" name="reset" />&nbsp;
-		{S_FORM_TOKEN}
-	</fieldset>
-	</form>
-
-<!-- ELSE -->
-
-	<!-- IF NUM_REMOVED -->
-		<div class="notice">
-			<p>{RULE_REMOVED_MESSAGES}</p>
-		</div>
-	<!-- ENDIF -->
-
-	<!-- IF NUM_NOT_MOVED -->
-		<div class="notice">
-			<p>{NOT_MOVED_MESSAGES}<br />{RELEASE_MESSAGE_INFO}</p>
+{% include 'ucp_header.html' %}
+
+{% if not PROMPT %}
+	{% include 'ucp_pm_message_header.html' %}
+{% endif %}
+
+{% if messagerow %}
+	<div class="pm-main">
+		<div class="pm-left-panel">
+			<ul class="pm-left-primary">
+				{% for message in messagerow %}
+					<li class="message-row {{ message.S_ROW_COUNT is odd ? 'row1' : 'row2' }} {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'msg-unread' }}">
+						<span class="left-inner">
+
+							<a href="{message.U_VIEW_PM}" class="msg-title {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'title-unread' : 'title-read' }}">{message.SUBJECT}</a><br><br>
+							<span class="pm-left-secondary">
+
+								{% if S_SHOW_RECIPIENTS %}
+								<span class="row-msg-author">
+									{{ lang('MESSAGE_TO') }} {message.RECIPIENTS}
+
+								{% else %}
+									{message.MESSAGE_AUTHOR_FULL}
+								</span>
+								{% endif %}
+
+								{% if message.PM_CLASS == "pm_foe_colour" %}
```

Comment:
I do not know about this the icon choices are not meaningful and this just makes no sense. I also do not think the friend is useful or used by anyone, more so for so maybe if they are foe then change the border color?.

Reply:
Alright, so will this be given preference over unread message border or vice versa?

Reply:
I would think trump it cause at least for me I want to ignore those

Reply:
I meant that when this conflicts with the red border for unred messages which gets the !important.
We can probably skip the icons entirely

Reply:
@hanakin and what about my original idea to decrease opacity of such item?

Reply:
First never use important you just add the more trump class last in the html on the tag and after the classes it needs to trump in the css. 

@senky Actually that may work. Sort of like a disabled and we may actually just be able style. It would be kind of difficult to pull off as is. we would need to move the bg class to the main item class using pseudo even/odd trick which is how it should be and not done via php.

What I suggest you do going forward is fork the codepen I made and use that to play around and get everything correct the way we want it and then just port it all over as is. This is much easier to prototype/review and work with not to mention faster. You can then just post update notifications here as you make version changes

Reply:
> just add the more important class last in the html on the tag
@hanakin could you please explain this part?
Is it needed just as a part of code style?
Because the position of classes in HTML seems to not change anything (see my previous question to you [here](https://github.com/phpbb/phpbb/pull/5222#discussion_r194224989)).

Reply:
Sure, this is how it looks with lower opacity:

https://codepen.io/senky/pen/eKRbMJ?editors=1100

I think it's still missing something to be clear to the users...

Reply:
I agree, 
maybe black coloured/underlined font for foe usernames

Reply:
@lavigor 

both css and html changes are required. you always put trumps after what it should trump in your css file and after the class in your html tag

https://codepen.io/hanakin/pen/yEaJQX?editors=1100

btw @senky @akbarhashmi I updated my codepen with foe and marked...